api/handlers/vouchers.go

```diff
@@ -49,65 +53,50 @@ func GetVoucherHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	ownerKeys, err := db.FetchOwnerKeys()
-	if err != nil {
-		slog.Debug("Error querying owner_keys", "error", err)
-		http.Error(w, err.Error(), http.StatusInternalServerError)
+	var ovBytes bytes.Buffer
+	if err := pem.Encode(&ovBytes, &pem.Block{
```

Comment:
This can be encoded directly to the responsewriter (after headers have been written) instead of a buffer.